### PR TITLE
KAFKA-4115: grow default heap size for connect-distributed.sh to 1G

### PR DESCRIPTION
**Pull request by shikhar**

### CODE REVIEW

**Comment by shikhar** [comment]
@ewencp for quick review

File: bin/connect-distributed.sh [hunk 1]
```diff
@@ -26,6 +26,10 @@ if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
     export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/connect-log4j.properties"
 fi
 
+if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
+    export KAFKA_HEAP_OPTS="-Xmx1G -Xms1G"
```
Comment (shikhar) [inline bin/connect-distributed.sh:30]: @ewencp perhaps we should make it `-Xmx1G -Xms256M`?

Or, maybe 512M as default max is a more conservative bump for now?

**Comment by ewencp** [comment]
@shikhar Let's get this resolved since it should be a really simple one. I like the idea of just using a smaller `Xms` setting, which lets us continue to support a lower memory usage profile when it is warranted, but still grow when necessary. Unless there's some objection to making these different (perhaps a startup or allocation pause concern), I think this makes sense.

The only other thing I'd suggest is also adjusting the `connect-standalone` settings as well.

**Comment by ewencp** [comment]
@gwenshap Thoughts on this? We've still not increased this for connect, but I'm not sure what we want to do for min vs max as we have comments in this thread already. More generally, is it perhaps time to revisit the setting in `kafka-run-class.sh`? Perhaps this is an issue beyond just Connect?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6225/
Test FAILed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6241/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by hachikuji** [comment]
Closing since this was resolved by #4213.

**Pull request closed without merging**